web/src/engine/predictive-text/worker-thread/src/main/correction/alignment-helpers.ts

```diff
@@ -236,7 +236,7 @@ export function computeAlignment(
     3
   );
 
-  let editPath = mapping.editPath();
+  let editPath = mapping.editPath()[0].map(t => t.op);
```

Comment:
Huh.  So the Sentry report was raised here - no edit path successfully built.  

https://keyman.sentry.io/share/issue/49be385e6aec40d5865c8262cfd54377/

> ```
> Error: Uncaught TypeError: Cannot read properties of undefined (reading 'map')
>  	at ? (blob:null/1ebfd692-da42-4b3d-88eb-aeca213e0ea3:3379:41)
> ```

The `.map` substring starts at char index 41 within its line.

That... shouldn't be possible, so there must be some sort of logic error in this PR's changes.  Wish I knew what the context was around the time of the error; that'd likely help here.